SU-SWS/decanter-web
# ADAPTDCTR-35, ADAPTDCTR-42 | @jdwjdwjdw | Adding Tailwind Plugins page, update favicon

# READY FOR REVIEW

# Summary
- [ADAPTDCTR-35](https://stanfordits.atlassian.net/browse/ADAPTDCTR-35): DOC | Add content to decanter v7 site - Tailwind Plugins
- [ADAPTDCTR-42](https://stanfordits.atlassian.net/browse/ADAPTDCTR-42): DEV | Update site favicon

# Needed By (Date)
- Retro review when convenient

# Urgency
- Normal

# Steps to Test

1. Checkout branch
2. Review [Tailwind Plugins](https://deploy-preview-139--dev-decanter-v7.netlify.app/for-developers/tailwindcss-plugins) page
3. Review site favicon and confirm that it has been updated to the Stanford logo favicon
4. Review code 🥞 

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [ADAPTDCTR-35](https://stanfordits.atlassian.net/browse/ADAPTDCTR-35): DOC | Add content to decanter v7 site - Tailwind Plugins
- [ADAPTDCTR-42](https://stanfordits.atlassian.net/browse/ADAPTDCTR-42): DEV | Update site favicon

[ADAPTDCTR-35]: https://stanfordits.atlassian.net/browse/ADAPTDCTR-35?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ADAPTDCTR-42]: https://stanfordits.atlassian.net/browse/ADAPTDCTR-42?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ADAPTDCTR-35]: https://stanfordits.atlassian.net/browse/ADAPTDCTR-35?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ADAPTDCTR-42]: https://stanfordits.atlassian.net/browse/ADAPTDCTR-42?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
